backend-agent/cli.py

```diff
@@ -379,6 +379,62 @@ def run(args):
               file=sys.stderr)
 
 
+def run_all_logic(args):
+    """Run all LLM attacks with specified target and evaluation models."""
+    # Load JSON spec
+    if not args.file:
+        print(
+            'No file given. Enter specification using stdin.',
+            file=sys.stderr
+        )
+        input_data = ''
+        for line in sys.stdin:
+            input_data += line
+            if line == '\n':
+                break
+        if not input_data:
+            print(
+                'Specify a path to a JSON spec or provide it via stdin.',
+                file=sys.stderr
+            )
+            return
+        spec = json.loads(input_data)
+    else:
+        with open(args.file, 'r') as f:
+            spec = json.load(f)
+
+    if 'attacks' in spec:
+        suite = AttackSuite.from_dict(spec)
+        suite.set_target(args.target_model)
```

Comment:
in case we want to support attack_model and eval_model in parameters, we need to inject them here